modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -317,6 +323,230 @@ public void testFactoryConfigurationWithUnderline() throws Exception {
 		_testFactoryConfiguration(CharPool.UNDERLINE);
 	}
 
+	@Test
+	public void testFindConfigurationWithFelixFileName() throws Exception {
+		String configurationPid = _CONFIGURATION_PID_PREFIX.concat(
+			".testFindConfigurationWithFelixFileName");
+
+		_configurationPath = Paths.get(
+			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR,
+			configurationPid.concat(".config"));
+
+		_configuration = _createConfiguration(
+			configurationPid, StringPool.BLANK);
+
+		Configuration[] configurations = _findExistingConfigurations(
+			_configurationPath);
+
+		Assert.assertNotNull(configurations);
+		Assert.assertEquals(
+			Arrays.toString(configurations), 1, configurations.length);
+		Assert.assertEquals(_configuration, configurations[0]);
+	}
+
+	@Test
+	public void testFindConfigurationWithFelixFileNameWhenUsingSymlink()
+		throws Exception {
+
+		String configurationPid = _CONFIGURATION_PID_PREFIX.concat(
+			".testFindConfigurationWithFelixFileNameWhenUsingSymlink");
+
+		_configurationPath = Paths.get(
+			FileUtil.createTempFileName(configurationPid, "config"));
+
+		Path symlinkConfigurationPath = Paths.get(
+			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR,
+			configurationPid.concat(".config"));
+
+		try {
+			Files.createSymbolicLink(
+				symlinkConfigurationPath, _configurationPath);
+
```

Comment:
Hi @ericyanLr,  `Files.createSymbolicLink` will throw java.nio.file.FileSystemException and the test can't be passed in 
windows (my os is window 10, jdk 1.8.0_152). Refer to the link https://bugs.openjdk.org/browse/JDK-8218418 and it seems it is a limitation for windows. I didn't find one way to fix it yet. Could you please check this if we can fix it for windows? If we can't find one way to fix it, we may skip the test in windows.

Reply:
Hi @yuhai , 

Based on the provided link, a fix was added to JDK 13.
- But from the comments, it's unclear whether it's supported in all versions of Windows and whether you need to enable certain features in Windows.

As for a potential workaround when manually testing, it looks like running the test as an Administrator might work:
- https://stackoverflow.com/questions/19230535/create-windows-symbolic-link-with-java-equivalent-to-mklink
- https://www.logicbig.com/how-to/code-snippets/jcode-java-io-files-createsymboliclink.html
  > Note that the Java code using this method will fail if run without Admin access. To run with Admin access, for example on windows, right click on cmd.exe and select 'Run as administrator' and run the Java program from there.
    - **Note:** I don't use Windows, so I haven't tested if this actually works.

But ya, I don't know of a way to fix this. I also did a quick search and it doesn't look like we have any tests that use symlinks, so I'm not sure how we would like to handle this.

Reply:
Hi @ericyanLr,

Actually, I had tried to run the test as an Administrator, but it didn't work for me.